static/blockchains/terra/wallets.json

```diff
@@ -0,0 +1,128 @@
+{
+  "terra1g6pwngzfj73wnxm7y5gae0thylphugumc2k48z": {
+    "name": "Hermes Protocol"
+  },
+  "terra15w7ntdhcyfce3twyj5s73gm536q7vn8yf5vp6n": {
+    "name": "Momentum One"
+  },
+  "terra1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8pm7utl": {
+    "name": "Core Distribution"
+  },
+  "terra1fl48vsnmsdzcv85q5d2q4z5ajdha8yu3nln0mh": {
+    "name": "Core Delegation"
+  },
+  "terra1tygms3xhhs3yv487phx3dw4a95jn7t7l8l07dr": {
+    "name": "Core Undelegation & Redelegation"
+  },
+  "terra1ur4davql0wc097frk0fskf6ycce4rwskc49lrr": {
+    "name": "Chinoman10"
+  },
+  "terra15hvzj53lvmtnnj5q8hqruwuff6vp4x77dqmu8c": {
+    "name": "Papi"
+  },
+  "terra18ruqkccl5tp493uhvra0u6jylrzq8t8dv5qs4c": {
+    "name": "Vini"
+  },
+  "terra19sft3gk3enekjkw34zgmnrqq2zcvt2t6ueqhac": {
+    "name": "MC"
+  },
+  "terra17tyjucdrq0qhu3gzxc2u04r5he8je35velus0e": {
+    "name": "Michael"
```

Comment:
I think "Michael" is a too generic name...
Let's remove this wallet from this list until a more unique name can be used.